core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStore.java

```diff
@@ -507,4 +509,29 @@ private int getDirIndexOfBlock(long blockId) {
         .getDir()
         .getDirIndex();
   }
+
+  private void removeBlockInternal(long sessionId, long blockId, long timeoutMs)
+      throws IOException {
```

Comment:
Should us replace the `IOException` to the concrete exception, something like `DeadlineExceededException `

Reply:
Yes, if concrete exceptions can be specified.

Reply:
The delete func in PageStore throws IOException (pageInfo.getLocalCacheDir().getPageStore().delete(pageId);), so we still need to throw IOException here.

Reply:
It seems make sense.